standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -592,7 +592,7 @@ struct ColumnStatistics {
 2: required list<ColumnStatisticsObj> statsObj,
 3: optional bool isStatsCompliant, // Are the stats isolation-level-compliant with the
                                                       // the calling query?
-4: required string engine
+4: optional string engine
```

Comment:
Why don't we keep it required and set the default engine (Hive) if metadata doesn't have it?

Reply:
in older dumps ..it is not set ..so if we keep it required ..it will fail during deserialization while reading the old dump files ...

Reply:
ok got it.